pkgs/build-support/bintools-wrapper/default.nix

```diff
@@ -108,7 +108,8 @@ let
   coreutils_bin = optionalString (!nativeTools) (getBin coreutils);
 
   # See description in cc-wrapper.
-  suffixSalt = replaceStrings ["-" "."] ["_" "_"] targetPlatform.config;
+  suffixSalt = replaceStrings ["-" "."] ["_" "_"] targetPlatform.config
+    + lib.optionalString (targetPlatform.isDarwin && targetPlatform.isStatic) "_static";
```

Comment:
It would affect other platforms if they had a special variable like `DEVELOPER_DIR` that could only resolve to one path, but only Darwin does that currently.